deploy/chart/templates/instrumentedapplications.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.8.0
+  creationTimestamp: null
+  name: instrumentedapplications.logz.io
+spec:
+  group: logz.io
+  names:
+    kind: InstrumentedApplication
+    listKind: InstrumentedApplicationList
+    plural: instrumentedapplications
+    singular: instrumentedapplication
```

Comment:
We can also add under `names` the field `shortNames` that can have an array of short names for the resource (like when instead of daemonset you can use ds)